examples/nodejs/doc-examples-js-apis.ts

```diff
@@ -806,6 +818,52 @@ async function example_API_RefreshAuthToken(authClient: AuthClient) {
   }
 }
 
+async function example_API_TopicPublish(topicClient: TopicClient) {
+  const result = await topicClient.publish('test-cache', 'test-topic', 'test-topic-value');
+  if (result instanceof TopicPublish.Success) {
+    console.log("Value published to topic 'test-topic'");
+  } else if (result instanceof TopicPublish.Error) {
+    throw new Error(
+      `An error occurred while attempting to publish to the topic 'test-topic' in cache 'test-cache': ${result.errorCode()}: ${result.toString()}`
+    );
+  }
+}
+
+async function example_API_TopicSubscribe(topicClient: TopicClient) {
+  const result = await topicClient.subscribe('test-cache', 'test-topic', {
+    onError: () => {
+      return;
+    },
+    onItem: () => {
+      console.log("Publishing values to the topic 'test-topic'!");
+      return;
+    },
+  });
+  if (result instanceof TopicSubscribe.Subscription) {
+    console.log("Successfully subscribed to topic 'test-topic'");
+
+    // Wait for stream to start.
+    setTimeout(() => {
+      console.log('Waiting for the stream to start');
+    }, 2000);
```

Comment:
This shouldn't be necessary, have you tried without it?

Reply:
I tried it locally, not required